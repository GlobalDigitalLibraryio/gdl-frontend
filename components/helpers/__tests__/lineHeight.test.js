// @flow
/**
 * Part of GDL gdl-frontend.
 * Copyright (C) 2017 GDL
 *
 * See LICENSE
 */

import lineHeight from '../lineHeight';

test('Returns pixel values', () => {
  expect(lineHeight({ lineHeight: 18 })).toMatchSnapshot();
});

test('Returns string values', () => {
  expect(lineHeight({ lineHeight: '3em' })).toMatchSnapshot();
});

test('Returns responsive values', () => {
  expect(lineHeight({ lineHeight: [18, 20] })).toMatchSnapshot();
});
