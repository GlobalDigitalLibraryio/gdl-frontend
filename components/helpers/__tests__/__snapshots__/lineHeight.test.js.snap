// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Returns pixel values 1`] = `
Object {
  "lineHeight": "18px",
}
`;

exports[`Returns responsive values 1`] = `
Object {
  "@media screen and (min-width: 40em)": Object {
    "lineHeight": "20px",
  },
  "lineHeight": "18px",
}
`;

exports[`Returns string values 1`] = `
Object {
  "lineHeight": "3em",
}
`;
